### PR TITLE
Display basic help information when no plugins are loaded

### PR DESCRIPTION
**Pull request by mjeronimo**
Fixes: #267

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

### CODE REVIEW

File: rqt_gui/src/rqt_gui/main.py [hunk 1]
```diff
@@ -53,10 +53,18 @@ def main(self, argv=None, standalone=None, plugin_argument_provider=None):
...
-
-        return super(
-            Main, self).main(argv, standalone=standalone,
-                             plugin_argument_provider=plugin_argument_provider)
+        help_text = """<p><b>rqt</b> is a GUI framework that is able to load various plug-in tools
+as dockable windows. There are currently no plug-ins selected. To add plug-ins, select items from
+the <b>Plugins</b> menu.</p>
+<p>You may also save a particular arrangement of plug-ins as a <i>perspective</i> using the
+<b>Perspectives</b> menu.
+<p>See <a href="http://wiki.ros.org/rqt/">the rqt Wiki page</a> for more information</p>
```
Comment (clalancette) [inline rqt_gui/src/rqt_gui/main.py]: So I'd really like not to point to the ROS 1 wiki anymore.  However, we don't have great documentation about RQt on https://docs.ros.org .  My suggestion here is to just remove this sentence, and then open a new issue on https://github.com/ros2/ros2_documentation to port RQt documentation over there.  Once that is in, we can always revisit this and add the link back.

**Comment by mjeronimo** [comment]
CI (build: `--packages-above-and-dependencies rqt_gui_cpp` test: `--packages-select rqt_gui_cpp`)

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/16585/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/11167/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/16971/)

File: rqt_gui/src/rqt_gui/main.py (hunk 1 above)
Comment (mjeronimo) [reply rqt_gui/src/rqt_gui/main.py]: Removed the link to the ROS 1 Wiki and added an issue to the ROS 2 documentation: https://github.com/ros2/ros2_documentation/issues/2415

**Review by clalancette — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**